src/main/java/org/opensrp/service/StockService.java

```diff
@@ -185,6 +216,114 @@ public void deleteStock(Long id) {
 	  return allStocks.findStocksByLocationId(servicePointId);
 	}
 
+	public CsvBulkImportDataSummary convertandPersistInventorydata(List<Map<String, String>> csvStocks, String userName) {
+		int rowCount = 0;
+		CsvBulkImportDataSummary csvBulkImportDataSummary = new CsvBulkImportDataSummary();
+		FailedRecordSummary failedRecordSummary;
+		List<FailedRecordSummary> failedRecordSummaries = new ArrayList<>();
+		Inventory inventory;
+		Integer totalRows = csvStocks.size();
+		Integer rowsProcessed = 0;
+
+		try {
+			failedRecordSummaries = validateInventoryData(csvStocks);
+		}
+		catch (ParseException e) {
+			logger.error("Exception occurred : " + e.getMessage(), e);
+			failedRecordSummary = new FailedRecordSummary();
+			List<String> validationError = new ArrayList<>();
+			failedRecordSummary.setRowNumber(rowCount);
+			validationError.add("Unknown error occurred");
```

Comment:
Since we know this is a parse exception can we at least reflect that and provide more details

Reply:
Updating error message to Parse Exception occurred.